bindings/cpp/session.cpp

```diff
@@ -1784,12 +1789,18 @@ static async_backend_control_result update_backend_status(const backend_status_p
 	backend_control->ids_count = params.ids.size();
 	backend_control->defrag_level = params.defrag_level;
 	backend_control->delay = params.delay;
+	backend_control->chunks_dir_len = params.chunks_dir.size();
 
 	if (!params.ids.empty()) {
 		data_pointer tmp = data.skip<dnet_backend_control>();
 		memcpy(tmp.data(), params.ids.data(), params.ids.size() * sizeof(dnet_raw_id));
 	}
 
+	if (!params.chunks_dir.empty()) {
+		auto tmp = data.skip<dnet_backend_control>().skip(params.ids.size() * sizeof(dnet_raw_id));
```

Comment:
Maybe it would be more elegant if we develop `data_pointer::skip<T>(size_t count = 1)` and use it here to calculate the end of expression.

Reply:
I thought about it but it is located in public header. Let's make it in a separate PR in backward-compatible way.